packages/-ember-data/tests/acceptance/relationships/belongs-to-test.js

```diff
@@ -278,11 +278,13 @@ module('async belongs-to rendering tests', function(hooks) {
       petOwner = await tweety.get('petOwner');
       assert.equal(petOwner.get('name'), 'Pete');
 
+      let goofyInternalModel = goofy._internalModel;
       await goofy.destroyRecord();
-      assert.ok(goofy.isDeleted);
+      assert.ok(goofyInternalModel.isDeleted, 'record is deleted');
```

Comment:
These changes were required since the original model does not have that property anymore.  The blocking PR helps get us closer to ensuring users can still call `goofy.isDeleted` and returning `true`.

https://github.com/emberjs/data/pull/7226#issuecomment-643552364

Reply:
I suspect this was actually an issue with the `DSModel` types that I've fixed in #7471 (but which would also have been improved by the native model class refactor since we could start using it for type information directly). We can probably resume work here, though the native model refactor is close!

Reply:
This is also improved in #7475